api/tests/features/environments/update.feature

```diff
@@ -34,3 +34,11 @@ Feature: Update my environment
     And should have a field "id" with "e0000001-0000-0000-0000-000000000000"
     And should have a field "owner" with "RwbFXSyjZ6F3HJ3Qc5CGUNMdSX8f3m79@clients"
     And should have a field "title" with "UpdateEnvironment A"
+
+  Scenario: Checking URL format conforms when updating an environment
+    Given I am logged in
+    When I make a "POST" request to "/environments" with:
```

Comment:
For update, the method type is `PATCH`.

`GET` = without ID is for a list
`GET` = with ID is for a single item
`POST` = create
`PATCH` = update

I hope that helps 🤓